config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -58,25 +58,25 @@ periodics:
     testgrid-dashboards: sig-node-containerd
     testgrid-tab-name: containerd-build
     description: "builds development in progress branch of upstream containerd"
-- name: ci-containerd-build-1-5
+- name: ci-containerd-build-1-6
   interval: 30m
   labels:
     preset-service-account: "true"
   spec:
     containers:
       - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230222-b5208facd4-master
         args:
-          - --repo=github.com/containerd/containerd=release/1.5
+          - --repo=github.com/containerd/containerd=release/1.6
           - --root=/go/src
           - --upload=gs://kubernetes-jenkins/logs
           - --scenario=execute
           - --
-          - --env=DEPLOY_DIR=release-1.5
+          - --env=DEPLOY_DIR=release-1.6
           - /go/src/github.com/containerd/containerd/test/build.sh
   annotations:
     testgrid-dashboards: sig-node-containerd
-    testgrid-tab-name: containerd-build-1.5
-    description: "builds release/1.5 branch of upstream containerd"
+    testgrid-tab-name: containerd-build-1.6
+    description: "builds release/1.6 branch of upstream containerd"
```

Comment:
@SergeyKanzhelev the args below and annotations don't match up exactly but probably ok since it's just build, right?